### ISSUE TITLE
Deploy owncloud, but js error

### ISSUE THREAD

**Issue by gmagic10** [owncloud/core#28281]
I've deploy owncloud on my machine.
CentOS 6 and nginx
I can create the admin account.
But when I login to the web. It report js error
`Uncaught TypeError: this.$tagsField.select2 is not a function`

I can't use the web.

Desktop client is working ok.

**Comment by kdslkdsaldsal** [comment]
@gmagic10 Please note this is a bugtracker of ownCloud. Issues like this are mostly caused by wrong server setups (especially as you're using nginx). Please close here and ask for support at https://central.owncloud.org/

Only once it is sure that this is a bug in ownCloud you can re-create a new issue here. If doing so you should make sure that you're filling out the issue template shown when creating a new issue (currently even the basic info about your environment is missing).
